build/builddeps/builddeps.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
per https://github.com/kubernetes/community/pull/3493 and https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module, you can use a build tag to import these dependencies, but never build them. In the community repo, I just created a `hack/tools.go` with a package named `tools`, and a build tag of `// +build tools` and it worked like a charm.

Reply:
the build tag is a nice tweak, will update

Reply:
I learned all about modules today and @cblecker beat me to this :)